Athena's Wrath/Athena's Wrath/main.swift

```diff
@@ -0,0 +1,355 @@
+            }
+            //won't print if statement for a after choosing b !!!!!
+            run = false
+        }
+    }
+    else if userChoice == 3{
+        print("Don't hide! You can do it!")
+        
+        decision = readLine()
+        
+        userChoice = Int(decision!) ?? 1
+        userChoice = Int(decision!) ?? 2
+        userChoice = Int(decision!) ?? 3
+    }
+} while run == true
+
+sleep(1)
+
+print("""
+
+Mist surrounds you. You can hear the a cacophany of hisses coming towards you. The mist suddenly clears and you see the giant snake-headed woman looming looming over you.
+""")
+sleep(1)
+
+print("""
+            @ .. . ,.,,... ,,,. ., ..,.  *  /,. .     ..*.  . .  ..   @
+            @.  .....,.* .,,...,. //@%@         ..  .,.     ,. .,,.   @
+            @  ,****...  . ,    .,(,,/.,   . ..  .,  ,. ..  .,...,,,,.@
+            @... ...*#%, .,,.. ./(#,(/&,*/*,*.  ., ,,. ... ..  ,.. .,.@
+            @     .   .,#(... .,*#%...*,....... *. *,  ... ...    ....@
+            @,,.,*.     ,,# ......   ,,.......  ,. */. .   ,,.        @
+            @..            * .,,%,.*,,&/(%#/*.. *, ,/(.   .,,.../     @
+            @      . ..     #*/*(/%*&%&&&&(*,. **.*/(%,...*,*,.     @
+            @ . .   /.,    *.((((#%(%&&(&@&&/*,  **..,##.,,*.....   @
+            @.,. .  ,,     **/(&%(,/#%(%&&@&@&&%#/*...*//*..*,.*(((/..@
+            @.   . .  .,*.*/##&&&&%%(%&&&&&%*....,,,.,/*....,.. .... *@
+            @.  .  .  ...    /#&&&&%**%%/. ...     .*/..  ,.    ,,.   @
+            @...      ,,..,(    ,(###(,,,    &**..,*(,,,..*,.   .,   ,@
+            @     ... *,*,.,/*,.**(&&%(/*,,,(/.,*//##((/#*,,.,,..,,,,.@
+            @ .   ... ,///////(#((%&@#%&%((//(#(#(*  .    .. .. ..  @
+            @ ,    ... *(#%&&&&&%#%&@&%%&@@@@@@##(*  ..,     ...*,,** @
+            @. .. .... ./#%&&&&&%#%&&&%%&@@@@(,/  ,//.. . . .*,,  . . @
+            @,   ,......*(#&&&&&%#%&@%##%&&&&*. /,*** ,  .../*  ,,,,. @
+            @..,,,,....../##%&&#%@@#((%&&&&//////*.  //.( ,  .,.,.@
+            @*///*,*. .  .*(#%%%#/,,*,(&&&&&%%%#((///.    ((.,(//, ,,.@
+            @ ..*(##%(. ...*((###((//%&&%%%%%#(((//*.  *.    (//** .*,@
+            @  ..,*. *%%*** *///.....   ...*((((//*.   *  **     .....@
+            @         ..*%,*.,*/( ,*,,/. *%((////*.    *  (*,      ..,@
+            @,  ,,/.     .*#,. *(/***,*/####.,,...     ( .*,//*,  .,,.@
+            @..*,*.,*.   .,./  .,(#%&&%%%(*#**,***.  (/**,/, .,,......@
+            @..../* ...    .,.   ./#%###.*(/**,.   **/,..   . .  , . .@
+            @..... ,. ... .  /     .... .%,,..   ,,.    .    ,* . .   @
+            @.,*,,,.  ..  .  /   .   . .....    *%*   ., .    . . . ..@
+""")
+sleep(2)
+print("""
+MEDUSA: "Hello, Athena; We meet again. It wasn't enough that you cursed me to look like this hideous creature, was it? I was a better woman than you then, and I still am. That is why I will give you a chance to flee, but only if you answer my riddle correctly. Are you ready?
```

Comment:
Remember that the user needs to be guided towards what choices are available, in this case, "yes" or "no"